lmp/build.sh

```diff
@@ -46,6 +46,7 @@ su builder -c $HERE/bb-config.sh
 touch ${archive}/customize-target.log && chown builder ${archive}/customize-target.log
 touch ${archive}/bitbake_debug.log ${archive}/bitbake_warning.log && chown builder ${archive}/bitbake_*.log
 touch ${archive}/bitbake_global_env.txt ${archive}/bitbake_image_env.txt && chown builder ${archive}/bitbake_*_env.txt
+touch "${archive}/app-preload.log" && chown builder "${archive}/app-preload.log"
```

Comment:
nit, mind removing " from before and after the file name in order to be aligned with the lines above this one?

Reply:
ok, although maybe the above should be put into `""` :).

Reply:
done.